test/games/prime1/exporter/test_prime_game_exporter.py

```diff
@@ -15,7 +16,9 @@ def test_patch_game(mocker, tmp_path, use_echoes_models):
     mock_patch_iso_raw: MagicMock = mocker.patch("py_randomprime.patch_iso_raw")
     mock_asset_convert: MagicMock = mocker.patch("randovania.patching.prime.asset_conversion.convert_prime2_pickups")
     mocker.patch("randovania.games.prime1.exporter.game_exporter.adjust_model_names")
-    patch_data = {"patch": "data", 'gameConfig': {}, 'hasSpoiler': True, "preferences": {}, "roomRandoMode": "None"}
+    patch_data = {"patch": "data", 'gameConfig': {}, 'hasSpoiler': True, "preferences": {}, "roomRandoMode": "None",
+                  "randEnemyAttributes": EnemyAttributeRandomizer(1.1, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, False).as_json,
+                  "seed": 106827108}
```

Comment:
There's a mismatch in parameters actually sent to `randomprime` and the ones we expected to be. The actual ones has the `seed`.

Reply:
If I get rid of the "seed" parameter then I get "seed" is not defined when my function is called, so how do I access the actual seed?

Reply:
Also my enemy_attributes keeps being sent to "dangerous_settings" as None rather than using what I have here in patch_data